app/src/main/java/org/oppia/android/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsDownloadPermissionsViewModel.kt

```diff
@@ -16,9 +18,13 @@ class AdministratorControlsDownloadPermissionsViewModel(
   private val oppiaLogger: OppiaLogger,
   private val profileManagementController: ProfileManagementController,
   private val userProfileId: ProfileId,
-  deviceSettings: DeviceSettings
+  deviceSettings: DeviceSettings,
+  automaticallyUpdateTopic: Boolean,
 ) : AdministratorControlsItemViewModel() {
 
+  val visibility: LiveData<Boolean> get() = _visibility
+  private var _visibility = MutableLiveData(automaticallyUpdateTopic)
```

Comment:
Does this need to be a LiveData if it never changes?

Reply:
Addressed